tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -2121,37 +2114,6 @@ def test_multiple_threshold_resolve_is_reset_when_count_is_lower_than_min_thresh
         self.assert_trigger_exists_with_status(incident, trigger, TriggerStatus.ACTIVE)
         self.assert_action_handler_called_with_actions(incident, [])
 
-    def test_ensure_case_when_no_metrics_index_not_found_is_handled_gracefully(self):
```

Comment:
The total `count` and `crashed` count now have explicit aliases, and we no longer group by `session.status`, so there will not be any unexpected values for that tag in the query result.